core/domain/exp_services.py

```diff
@@ -277,17 +279,9 @@ def get_multiple_explorations_by_version(exp_id, version_numbers):
     explorations = []
     exploration_models = exp_models.ExplorationModel.get_multi_versions(
         exp_id, version_numbers)
-    error_versions = []
-    for index, exploration_model in enumerate(exploration_models):
-        try:
-            explorations.append(get_exploration_from_model(exploration_model))
-        except utils.ExplorationConversionError:
-            error_versions.append(version_numbers[index])
-
-    if error_versions:
-        raise Exception(
-            'Exploration %s, versions [%s] could not be converted to latest'
-            'schema version.' % (exp_id, ', '.join(map(str, error_versions))))
+    for _, exploration_model in enumerate(exploration_models):
+        explorations.append(get_exploration_from_model(exploration_model))
```

Comment:
Review again!

Reply:
Since we are passing an exp_id here we won't get an ExplorationConversionError. Also for any invalid exp_id the exceptions would be raised from gae_models.py

Reply:
`get_exploration_from_model` can raise `ExplorationConversionError`, right?

(For the case where the exploration_model.schema_version < latest_exploration_schema_version we will be running the conversion methods which can raise ExplorationConversionError.)

Reply:
done